services/server/src/server/services/utils/database-util.ts

```diff
@@ -632,3 +636,38 @@ export async function getSourcifyMatchAddressesByChainAndMatch(
     [chain, page * paginationSize, paginationSize]
   );
 }
+
+export async function updateContract(
+  pool: Pool,
+  { id, creation_bytecode_hash, runtime_bytecode_hash }: Tables.Contract
+) {
+  return await pool.query(
+    "UPDATE contracts SET creation_code_hash = $2, runtime_code_hash = $3 WHERE id = $1 RETURNING *",
+    [id, creation_bytecode_hash, runtime_bytecode_hash]
+  );
+}
+
+export async function updateContractDeployment(
+  pool: Pool,
+  {
+    id,
+    transaction_hash,
+    block_number,
+    txindex,
+    deployer,
+    contract_id,
+  }: Omit<Tables.ContractDeployment, "chain_id" | "address">
+) {
```

Comment:
Here, a return type would also be nice, since it cannot be inferred from the `query` function.

Reply:
Maybe we can fix all of the return types in another PR, because it is something that has to be done everywhere

Reply:
Okay that's fine with me. But we should create an issue then